tests/lac/precondition_relaxation_01.output

```diff
@@ -1,13 +1,13 @@
 
-DEAL::OK!
-DEAL::OK!
-DEAL::OK!
-DEAL::OK!
-DEAL::OK!
-DEAL::OK!
-DEAL::OK!
-DEAL::OK!
-DEAL::OK!
-DEAL::OK!
-DEAL::OK!
-DEAL::OK!
+DEAL::OK! 6.06176 14.6813 6.06176 14.6813 
+DEAL::OK! 10.5221 19.3228 10.5221 19.3228 
+DEAL::OK! 15.3680 24.2003 15.3680 24.2003 
+DEAL::OK! 5.62500 14.2615 5.62500 14.2615 
+DEAL::OK! 9.85490 18.6445 9.85490 18.6445 
+DEAL::OK! 14.3212 23.1484 14.3212 23.1484 
+DEAL::OK! 5.06250 13.7224 5.06250 13.7224 
+DEAL::OK! 8.97855 17.7551 8.97855 17.7551 
+DEAL::OK! 12.9800 21.7988 12.9800 21.7988 
+DEAL::OK! 6.18750 14.8024 6.18750 14.8024 
+DEAL::OK! 10.7121 19.5161 10.7121 19.5161 
+DEAL::OK! 15.6708 24.5042 15.6708 24.5042 
```

Comment:
Let's make this output more than 10 digits. Otherwise, numdiff might trigger accidentally.

Reply:
Good idea, I adjusted the test and its output.